README.md

```diff
@@ -36,7 +36,7 @@ If you'd like to follow along with this tutorial on your own machine, you'll nee
 3. [Flux CD](https://fluxcd.io/flux/cmd/) command-line tool installed.
 4. A Kubernetes cluster running.
 
-In this tutorial, you'll use [kind](https://kind.sigs.k8s.io). You may also use alternatives like [k3d](https://k3d.io/), cloud providers or others.
+In this tutorial, you'll use [kind](https://kind.sigs.k8s.io). You may also use alternatives like [k3d](https://k3d.io/), cloud providers, or others.
```

Comment:
mmh not sure about this one. @svx : wdyt ?

Reply:
You mean the comma? 😄 It was suggested by Grammar's. Otherwise, the text comes from Nico's previous PR I think

Reply:
I just double checked and yeah the comma here is an improvement!

Reply:
That's unexpected. ok :).